### PR TITLE
Convert all databases modules to python3 and 2.4 syntax

### PR DESCRIPTION
**Pull request by mscherer**
##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @mscherer for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: database/mysql/mysql_db.py [hunk 1]
```diff
@@ -289,8 +289,9 @@ def main():
     try:
         cursor = mysql_connect(module, login_user, login_password, config_file, ssl_cert, ssl_key, ssl_ca,
                                connect_timeout=connect_timeout)
-    except Exception, e:
-        if os.path.exists(config_file):
+    except Exception:
+        e = get_exception()
+        if oes.path.exists(config_file):
```
Comment (abadger) [inline database/mysql/mysql_db.py]: Typo here -- should remain `os.path.exists()`

**Comment by abadger** [comment]
Found a typo but otherwise the changes here are good.  Once the typo is fixed we can merge this.

**Comment by mscherer** [comment]
yeah, I shouldn't have done that after coming from party. I also rebased since we had a conflict

**Comment by abadger** [comment]
Thanks @mscherer !

**Pull request merged**